src/D2L.CodeStyle.TestAnalyzers/NUnit/CategoryAnalyzer.cs

```diff
@@ -53,7 +54,7 @@ internal sealed class CategoryAnalyzer : DiagnosticAnalyzer {
 					context: ctx,
 					bannedCategories: bannedCategories,
 					types: types,
-					syntax: ctx.Node as MethodDeclarationSyntax
+					syntax: (ctx.Node as MethodDeclarationSyntax)!

```

Comment:
nit: `SyntaxNodeAnalysisContext.Node` is already non-nullable. If this were a hard-cast would we be able to avoid needing to force non-null?

Reply:
Ah yeah sure. Technically a hard cast throws a different exception/earlier but the reason I used `!` is that in the context the cast will always succeed, so yeah I'll switch to hard cast